samples/scales/timeseries/line-point-data.html

```diff
@@ -0,0 +1,152 @@
+<!doctype html>
+<html>
+
+<head>
+	<title>Time Scale Point Data</title>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.13.0/moment.min.js"></script>
+	<script src="../../../dist/Chart.js"></script>
+	<script src="../../utils.js"></script>
+	<style>
+    canvas {
```

Comment:
Looks like there's some indentation here that should be switched from spaces to tabs